### ISSUE TITLE
Reduce the size of output Dll

### ISSUE THREAD

**Issue by surrealist** [RevensofT/.Net-Anarchy#1]
I think that My can remove out of the project. You may write Build script to directly compile only .il and .vb

**Comment by RevensofT** [comment]
Remove namespace My will 'cause issue to MSBuilder when compile .il, if you can do it with out an error then fork and pull requests, I will merge them to master branch soon as posible. :)

**Comment by surrealist** [comment]
Could you please try to build it using `vbc` and `ilasm` directly? I don't understand the purpose of .vb files, the real code is only the IL, isn't it?

**Comment by RevensofT** [comment]
VB part is for add reference by solution to other project, that make me can edit Anarchy project and effect on Example project without compile and reload Anarchy.dll on Example project(it's easy for anyone who clone project to edit and test on their own); If you use VS expression version I'm sure you will not ask me do that because VS will reject to add reference from any dll that not compile by VS(even you use msbuild to compile it, IDE won't support it), this is reason I'm move to VS Community version.

**Comment by surrealist** [comment]
> VS will reject to add reference from any dll that not compile by VS

I don't think so. I used to use `csc.exe` to compile .cs files for Dlls and it can be referenced by other projects via VS. I'd like to try `ilasm.exe` and `vbc.exe` on this project but I still don't understand all code enough to do it.

**Comment by RevensofT** [comment]
It's only when you use VS express version(I have this trouble with Vs VB.net Express 2012).
Other way to cut of namespace My is decompile a dll and remove all code under namespace My then recompile again; easily isn't it, so I leave to anyone who want to learn IL language do it for sake of training. :innocent:

**Comment by surrealist** [comment]
Oh, just know that limitation of VS Express edition.
